org.jacoco.core.test.validation.java7/src/org/jacoco/core/test/validation/java7/targets/TryWithResourcesTarget.java

```diff
@@ -32,6 +32,7 @@ public void close() {
 	 */
 	private static Object test() throws Exception {
 		nop(); // assertFullyCovered()
+
 		try ( // assertTry()
```

Comment:
Change of header revealed following strange behavior of ECJ `3.12.1` that we currently use in tests:

compilation of

```
/* comment with 20 lines */
import java.io.Closeable;

class Example {
    private static class Resource implements Closeable {
        @Override
        public void close() {
        }
    }

    private static void nop() {
    }

    private static void example() throws Exception {
        nop();
        try( // line 35
            Resource r1 = new Resource();
            Closeable r2 = new Resource();
            AutoCloseable r3 = new Resource();
        ) {
            nop();
        }
    }
}
```

results in bytecode with following `LineNumberTable`:

```
        line 34: 0
        line 36: 7
        line 37: 16
        line 3: 25
        line 40: 35
        line 41: 38
        line 42: 176
```

however if first comment contains one less/more lines than `20`, i.e. `try` keyword is not on line `35`, then `LineNumberTable` is different - for example when method starts at line `35`:

```
        line 35: 0
        line 36: 3
        line 37: 7
        line 38: 16
        line 39: 25
        line 41: 35
        line 42: 38
        line 43: 176
```

notice absence of entry for offset `3` in the first case.